quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/AddNetworkApiController.java

```diff
@@ -44,7 +50,13 @@
      */
     @ResponseBody
     @RequestMapping(value = "/api/v1/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
-    public HttpStatus processJson(@RequestBody Quadruple quadruple) {
+    public HttpStatus processJson(@RequestBody Quadruple quadruple, @RequestParam(value = "collectionid", required = true) String collectionId) {
```

Comment:
let's make the collection id a path variable, e.g. `/api/v1/collection/{collectionId}/network/add`